pkg/kn/commands/types.go

```diff
@@ -50,3 +55,11 @@ func GetConfig() (serving.ServingV1alpha1Interface, error) {
 	}
 	return client, nil
 }
+
+// GetCurrentNamespace returns current namespace from kubeconfig.
+func GetCurrentNamespace(kubeconfig string) (string, error) {
```

Comment:
can't really comment on the logic here as I don't know anything about how to extract a current context from the configuration. Looks complicated though ;-)

Reply:
Isn’t it one line in the `KUBECONFIG` file? Maybe my clusters are super simple...